### PR TITLE
Fixes vs2010 doesn't recognize host=x64

### PR DESCRIPTION
**Pull request by lygstate**
### This changes [[visible behavior/performance/documentation/etc.]]

The following changes are proposed:

- 
- 
-

### CODE REVIEW

File: src/kit.ts [hunk 1]
```diff
@@ -756,13 +756,18 @@ async function tryCreateNewVCEnvironment(inst: VSInstallation, hostArch: string,
...
       log.debug(` ${localize('generator.present', 'Generator Present: {0}', generatorName)}`);
       kit.preferredGenerator = {
         name: generatorName,
         platform: generatorPlatformFromVSArch[targetArch] as string || targetArch,
         // CMake generator toolsets support also different versions (via -T version=).
-        toolset: "host=" + hostArch
+        toolset
       };
     }
     log.debug(` ${localize('selected.preferred.generator.name', 'Selected Preferred Generator Name: {0} {1}', generatorName, JSON.stringify(kit.preferredGenerator))}`);
```
Comment (andreeis) [inline src/kit.ts:773]: How about we assign toolset as: 
    toolset: "host=" + kitHostTargetArch(hostArch, targetArch, majorVersion < 15)

Comment (lygstate) [reply src/kit.ts:773]: fine with that

Comment (andreeis) [reply src/kit.ts:773]: Great. Can you test that in your setup and if all is well commit an update? I'll approve it.

Comment (lygstate) [reply src/kit.ts:773]: toolset: "host=" + kitHostTargetArch(hostArch, targetArch, majorVersion < 15)

Why host= still here? that's should be removed

Comment (lygstate) [reply src/kit.ts:773]:
```
[main] 正在配置文件夹: xemu-with-lib 
[driver] 删除 e:/CI-Cor-Ready/xemu/xemu-with-lib/build-cmake/msvc_10_0_win32_x64_Debug/CMakeCache.txt
[driver] 删除 e:\CI-Cor-Ready\xemu\xemu-with-lib\build-cmake\msvc_10_0_win32_x64_Debug\CMakeFiles
[proc] 执行命令: "C:\Program Files\CMake\bin\cmake.EXE" --no-warn-unused-cli -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -He:/CI-Cor-Ready/xemu/xemu-with-lib -Be:/CI-Cor-Ready/xemu/xemu-with-lib/build-cmake/msvc_10_0_win32_x64_Debug -G "Visual Studio 10 2010" -T host=amd64 -A x64
[cmake] Not searching for unused variables given on the command line.
[cmake] CMake Error at CMakeLists.txt:17 (project):
[cmake]   Generator
[cmake] 
[cmake]     Visual Studio 10 2010
[cmake] 
[cmake]   given toolset specification
[cmake] 
[cmake]     host=amd64
[cmake] 
[cmake]   that contains invalid field 'host=amd64'.
[cmake] 
[cmake] 
[cmake] -- Configuring incomplete, errors occurred!
[cmake] See also "E:/CI-Cor-Ready/xemu/xemu-with-lib/build-cmake/msvc_10_0_win32_x64_Debug/CMakeFiles/CMakeOutput.log".

```

Comment (andreeis) [reply src/kit.ts:769]: Oh, I thought that the problem was using amd64 versus x64, not the host attribute itself. I see now, looks good. I am approving.

**Review by andreeis — APPROVED** [review]

**Pull request merged**